### DISCUSSION TITLE
Why the velocity embedding isn't consistent with the unspliced RNA proportions?

### DISCUSSION THREAD

**Discussion by danli349** [theislab/scvelo#783] [Q&A]
I think the unspliced RNA proportions should indicate the immature cell status and the velocity embedding should stem from high unspliced RNA proportion to low unspliced RNA proportion. Am I correct?
But why in my data, the velocity embedding isn't consistent with the unspliced RNA proportions?

![image](https://user-images.githubusercontent.com/33963919/146943652-1d5f9966-55df-485c-a978-31834285ddff.png)
![image](https://user-images.githubusercontent.com/33963919/146943707-04b81192-7ccb-48d9-9963-bd3bc488e2ea.png)

From the unspliced RNA proportions data, cluster 6 should be stem cell and cluster 3 should be its descendants, but the velocity embedding seems reversed.
Thanks

**Comment by ftucos** [comment]
Average spliced/unspliced RNA proportions are not informative of the trajectory. 
You should hand check the phase portrait of the top likelihood/dynamical genes explaining the predicted directions.
You can rank the genes by either `fit_likelihood`  if you are using a dynamical model or `velocity_r2`  if you are using a stochastic model:
`top_genes = adata.var['fit_likelihood'].sort_values(ascending=False)`
and then inspect the phase portraits:
`scv.pl.scatter(adata, top_genes[0:20], color='clusters', ncols=3, size=30)`

You must be aware of some pitfalls that could lead to a reverse/faulty estimation of the trajectory; give a look at [RNA velocity—current challenges and future perspectives](https://www.embopress.org/doi/full/10.15252/msb.202110282) and [Coordinated changes in gene expression kinetics underlie both mouse and human erythroid maturation](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02414-y)

**Reply by WeilerP** [reply]
Totally agree @ftucos. Just a small additional remark: I do understand where you are coming from @danli349, and you may be correct in an ideal world. However, increase/decreased counts of unspliced/spliced mRNA can be an experimental artefact. There are, for example, cases where genes that are known to be important for a given process are simply not captured sufficiently enough during sequencing to later on show up in RNA velocity analysis.
